app/javascript/src/components/Dashboard/Contacts/Table.jsx

```diff
@@ -56,7 +63,8 @@ const Table = ({ contacts }) => (
           dataIndex: "id",
           key: "id",
           width: "10%",
-          render: renderActionDropdown,
+          render: () =>
+            renderActionDropdown((setShowDeleteAlert = { setShowDeleteAlert })),
```

Comment:
I am not able to understand why you are doing an assignment while passing the arguments. Please put up a video and explain to me your thought process. I don't need a text reply. A video is must.

Reply:
Video: https://www.loom.com/share/d42c30056bee4ddcb1a7dddb89b577f8

Reply:
@anil-anand About positional args, please read: https://academy.bigbinary.com/learn-react/react-best-practices/structuring-code#:~:text=Positional%20parameters%20vs%20Named%20parameters

About the MenuBar code, watch: https://www.loom.com/share/aaf8c41dc3864949bea0a3026ac6158a

Reply:
Yedhin, I don't think this is a mistake.
In the code, there is a `Container` below the `MenuBar`, which holds an `Open MenuBar` / `Close MenuBar` button. 

#### Code:

![image](https://user-images.githubusercontent.com/54279092/192972262-47ac0e35-c4de-4bc0-8b9f-51620a362a8b.png)

#### Button in the UI:

![image](https://user-images.githubusercontent.com/54279092/192973089-2a94e4c5-0168-40c1-b940-26ef1eaec85b.png)

When I deleted the `Container` from the copy-pasted code, I forgot to delete the `div`. Sorry for the trouble!

Reply:
>Yedhin, I don't think this is a mistake.

Got it, In that case we don't need to raise any issue.

Reply:
Okay. After making the above changes to the file, can I proceed with deploying the application?

Reply:
Yes.

Reply:
You don't have to wait for my approval to merge PRs and continue with next tasks.